pkg/erofs/mediatype.go

```diff
@@ -0,0 +1,24 @@
+package erofs
+
+import (
+	"fmt"
+	"strings"
+)
+
+type ErofsCompression string
+
+const (
+	BaseMediaTypeLayerErofs = "application/vnd.stacker.image.layer.erofs"
+
+	LZ4HCCompression ErofsCompression = "lz4hc"
+	LZ4Compression   ErofsCompression = "lz4"
```

Comment:
You list the other types here, but we don;t check for them in the the `mkfs.erofs` output

Reply:
These are the original default ones. mkfs.erofs skipping these would be a major breakage.

Reply:
Any `mkfs.erofs` may not have those enabled; it seems strange to check for other flags, but not the "default" ones when the command itself indicates whether it's enabled or not.  Either drop the "check for features" function and use "original default ones", or check for all of them.

Reply:
Afflicts squashfs originally as well?
https://github.com/project-machine/atomfs/blob/main/squashfs/squashfs.go#L156

Reply:
Probably; I didn't review the original squashfs support.  But there's little reason to not do it correctly now and fix up the squashfs the same separately.

Reply:
Parity with functionality yes, that doesn't mean we have to keep the code the same; we can fix as we go.